model/src/main/kotlin/config/ScannerConfiguration.kt

```diff
@@ -65,5 +64,8 @@ data class ScannerConfiguration(
     /**
      * A list of globs that match file paths which are to be excluded from scan results.
      */
-    val ignoreFilePaths: List<String> = NON_LICENSE_FILENAMES
+    val ignoreFilePaths: List<String> = listOf(
+        "**/HERE_NOTICE",
+        "**/META-INF/DEPENDENCIES"
+    )
```

Comment:
I think I would prefer to have those in `default.conf` instead because I think it's easier for the user to compare/adapt the default.conf to a custom config file, and it's a single place to check for the default configuration. What do you think?
The default license filename patterns should also go there IMO.

Reply:
I think that's a reasonable default behaviour which is why I preferred to have them here.
What do you think about having it in both places, default.conf and here?

Reply:
Hm, am I right that the `default.conf` is currently actually empty?

Reply:
Yes, because so far there are either no defaults for the config options or they have not been moved to this file.
Putting them in both places means to places to maintain so I'd prefer just one of them, but we could keep the defaults here for now and then make a separate change to move all defaults to `default.conf` later.

Reply:
Agree, let's make it out of scope of this PR then.

Reply:
I've created an [issue](https://github.com/oss-review-toolkit/ort/issues/3522) for that so that we don't forget about it.